bin/ahn

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env ruby
 
 require 'adhearsion/cli'
-Adhearsion::CLI::AhnCommand.start
+# when script/ahn gets loaded it does AhnCommand.start
+unless Adhearsion::ScriptAhnLoader.load_script_ahn
```

Comment:
Maybe `Adhearsion::ScriptAhnLoader.load_script_ahn || Adhearsion::CLI::AhnCommand.start` is clearer in intent?